2025-05-01 15:40:47,860 - INFO - Starting task ID 5 for application 1176978325
2025-05-01 15:40:51,104 - INFO - 1176978325 24/03/2007 LMV,MCWG 10-11-2025   TRUE 1
2025-05-01 15:40:51,104 - INFO - 
2025-05-01 15:40:51,104 - INFO - Time reached: Starting the code...
2025-05-01 15:40:51,104 - INFO - 
2025-05-01 15:40:51,107 - INFO - 
2025-05-01 15:40:51,107 - INFO - Selected Proxy: {}
2025-05-01 15:40:51,107 - INFO - {}
2025-05-01 15:40:51,107 - INFO - start
2025-05-01 15:40:51,650 - INFO - making request to createTask
2025-05-01 15:40:53,517 - INFO - created task with id 400746241
2025-05-01 15:40:54,523 - INFO - making request to getTaskResult
2025-05-01 15:40:59,809 - INFO - task is still processing
2025-05-01 15:41:00,811 - INFO - making request to getTaskResult
2025-05-01 15:41:01,090 - INFO - task solved
2025-05-01 15:41:01,090 - INFO - captcha text 6TX8Sf
2025-05-01 15:41:01,090 - INFO - 6TX8Sf
2025-05-01 15:41:01,910 - INFO - Captcha success
2025-05-01 15:41:02,460 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:02,460 - INFO - start
2025-05-01 15:41:03,015 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:03,015 - INFO - start
2025-05-01 15:41:03,576 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:03,576 - INFO - start
2025-05-01 15:41:04,193 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:04,194 - INFO - start
2025-05-01 15:41:04,746 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:04,746 - INFO - start
2025-05-01 15:41:05,305 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:05,305 - INFO - start
2025-05-01 15:41:05,918 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:05,918 - INFO - start
2025-05-01 15:41:06,527 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:06,527 - INFO - start
2025-05-01 15:41:07,136 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:07,136 - INFO - start
2025-05-01 15:41:07,740 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:07,740 - INFO - start
2025-05-01 15:41:08,321 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:08,321 - INFO - start
2025-05-01 15:41:08,939 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:08,939 - INFO - start
2025-05-01 15:41:09,549 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:09,549 - INFO - start
2025-05-01 15:41:10,139 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:10,139 - INFO - start
2025-05-01 15:41:10,763 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:10,763 - INFO - start
2025-05-01 15:41:11,368 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:11,368 - INFO - start
2025-05-01 15:41:11,973 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:11,973 - INFO - start
2025-05-01 15:41:12,577 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:12,578 - INFO - start
2025-05-01 15:41:13,122 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:13,122 - INFO - start
2025-05-01 15:41:13,670 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:13,670 - INFO - start
2025-05-01 15:41:14,172 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:14,172 - INFO - start
2025-05-01 15:41:14,723 - INFO - name 'proceeddata' is not defined
2025-05-01 15:41:14,723 - INFO - start
2025-05-01 15:41:20,694 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,694 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,694 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,694 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,695 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,695 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,695 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,695 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,695 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,695 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,695 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,695 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,695 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,695 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,695 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,696 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,696 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,696 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,696 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,696 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,696 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,696 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,696 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,696 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,696 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,696 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,696 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,696 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,696 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,696 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,696 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,696 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,697 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,697 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,697 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,697 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,697 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,697 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,697 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,697 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,697 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,697 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,697 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,697 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,697 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,697 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,697 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,698 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,698 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,698 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,698 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,698 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,698 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,698 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,698 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,698 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,698 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,698 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,698 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,698 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,698 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,698 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,698 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,698 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,698 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,699 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,699 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,699 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,699 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,699 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,699 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,699 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,699 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,699 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,699 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,699 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,699 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,699 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,699 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,699 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,700 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,700 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,700 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,700 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,700 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,700 - ERROR - File "C:\Users\User\Downloads\DLPROJ\game.py", line 310, in <module>
2025-05-01 15:41:20,700 - ERROR - purp0_data = proceeddata
2025-05-01 15:41:20,700 - ERROR - NameError: name 'proceeddata' is not defined
2025-05-01 15:41:20,700 - ERROR - Traceback (most recent call last):
2025-05-01 15:41:20,715 - INFO - Task ID 5 status updated to Failed
